components/motor/fake/motor_test.go

```diff
@@ -60,15 +60,14 @@ func TestGoFor(t *testing.T) {
 
 	err = m.GoFor(ctx, 0, 1, nil)
 	allObs := obs.All()
-	latestLoggedEntry := allObs[len(allObs)-1]
-	test.That(t, fmt.Sprint(latestLoggedEntry), test.ShouldContainSubstring, "nearly 0")
+	test.That(t, fmt.Sprint(allObs), test.ShouldContainSubstring, "nearly 0")
+	// test.That(t, fmt.Sprint(latestLoggedEntry), test.ShouldContainSubstring, "nearly 0")
```

Comment:
I'd prefer not to merge any commented-out code. Can we either uncomment this line, or else delete it entirely?

Reply:
Fixed in 0216229b6